server/e2e_test_command.go

```diff
@@ -93,21 +91,6 @@ func (s *Server) handleE2ETest(ctx context.Context, commenter string, pr *model.
 		return e2eTestErr
 	}
 
-	ghStatusContext := ""
-	switch prRepoName {
-	case s.Config.E2EWebappReponame:
-		ghStatusContext = s.Config.E2EWebappStatusContext
-	case s.Config.E2EServerReponame:
-		ghStatusContext = s.Config.E2EServerStatusContext
-	}
-
-	ctx2, cancel := context.WithTimeout(context.Background(), defaultE2ETestTimeout*time.Second)
-	defer cancel()
-	err = s.waitForStatus(ctx2, pr, ghStatusContext, stateSuccess, 30*time.Second) // 30 seconds to consider GitHub's secondary rate limit
-	if err != nil {
-		return err
-	}
-
```

Comment:
Okay, I see, since success checks were already done in the [above LoC](https://github.com/mattermost/mattermost-mattermod/pull/308/files#diff-7047d41fe350835623854ec21ede87cc3c32f0f8842eba189be55bfb1d459bc6L86):
```go
isCIPassing, err := s.areChecksSuccessfulForPR(ctx, pr)
```

Reply:
I am still not sure, why I cannot query for the single docker build status. But atm figuring this out is not as pressing.